ui/racer/src/view/race.ts

```diff
@@ -20,7 +20,15 @@ export const renderRace = (ctrl: RacerCtrl) => {
   const tracks: VNodes = [];
   players.forEach((p, i) => {
     const isMe = p.name === myName;
-    const track = renderTrack(relative, isMe, bestScore, ctrl.boost, p, i);
+    const track = renderTrack(
+      relative,
+      isMe,
+      bestScore,
+      ctrl.boost,
+      p,
+      i,
+      hashCodeToVehicle(ctrl.race.id + i),
```

Comment:
this runs on every frame and for every player. Maybe they should be precomputed once, then reused.